linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
 mkdir ${td}
 cd ${td}
 wget ${srctgz}
-tar xfz ${file}
+tar xf ${file}
```

Comment:
I am wondering if we should be clever here and do something like

```
file x13ashtmlsrc_V1.1_B26.tar.gz | grep -q compressed
```

and based on the result call with `xfz` or `xf`, respectively.  We didn't plant major functionality bugs here on purpose -- I am *very* certain the files were in fact compressed a year ago.